### PR TITLE
flashfocus: init at 2.2.2 & pythonPackages.xpybutil: init at 0.0.6

### PR DESCRIPTION
**Pull request by Artturin**
Add flashfocus and a not yet packaged dependency
###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [X] other Linux distributions
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/xpybutil/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+buildPythonPackage rec {
+  pname = "xpybutil";
+  version = "0.0.6";
+
+  # Pypi only offers a wheel
+  src = fetchFromGitHub {
+    owner = "BurntSushi";
+    repo = "${pname}";
+    rev = "${version}";
```
Comment (jonringer) [inline pkgs/development/python-modules/xpybutil/default.nix:11]:
```suggestion
    repo = pname;
    rev = version;
```

**Review by jonringer — COMMENTED** [review]
otherwise LGTM

File: pkgs/development/python-modules/xpybutil/default.nix (hunk 1 above)
Comment (Artturin) [reply pkgs/development/python-modules/xpybutil/default.nix]: done

**Review by jonringer — APPROVED** [review]
LGTM
shows usage

```
https://github.com/NixOS/nixpkgs/pull/95757
4 packages built:
flashfocus python27Packages.xpybutil python37Packages.xpybutil python38Packages.xpybutil
```

**Comment by Artturin** [comment]
Thanks!

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -1917,6 +1917,8 @@ in
 
   fuzzel = callPackage ../applications/misc/fuzzel { };
 
+  flashfocus = python3Packages.callPackage ../misc/flashfocus { };
```
Comment (dotlambda) [inline pkgs/top-level/all-packages.nix:1920]: Don't use `python3Packages.callPackage` in `all-packages.nix`!
Can someone please fix that?

Comment (dotlambda) [reply pkgs/top-level/all-packages.nix:1920]: https://github.com/NixOS/nixpkgs/pull/117027

**Pull request merged**